x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/service/ElasticServiceAccounts.java

```diff
@@ -49,6 +49,12 @@ final class ElasticServiceAccounts {
                         ".elastic-connectors*"
                     )
                     .privileges("manage", "read", "write")
+                    .build(),
+                // Connector secrets. Enterprise Search can read and write to this index.
+                RoleDescriptor.IndicesPrivileges.builder()
+                    .indices(".connector-secrets*")
```

Comment:
I think we no longer need explicit permissions for connector-secrets index, as that's taken care of by the OriginSettingClient + privileges combo